index.html

```diff
@@ -7,7 +7,7 @@
 <body>
   <p>Open dev tools to try it out</p>
   <ul>
-    <li><a href="test/">Run unit tests in browser.</a></li>
+    <li><a href="http://localhost:9876/debug.html">Run unit tests in browser.</a></li>
```

Comment:
maybe this should use `window.location`? that way `localhost` and an ip address will both work.